### DISCUSSION TITLE
How do I use LLM to obtain standard JSON data?

### DISCUSSION THREAD

**Discussion by zswll2** [langgenius/dify#4311] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

**"text": "```json\n{\n  \"testl4\": {\n    \"ID\": \"双面胶带\",\n    \"A\": \"3M9448A\",\n    \"B\": \"0.14mm厚\"\n  }\n}\n```",**

The above is my request for llm to provide me with 3 keywords in the form of JSON data. However, what can be done to remove these symbols and obtain a standard JSON data?

### 2. Describe the feature you'd like to see

Standard JSON format

### 3. How will this feature improve your workflow or experience?

This will greatly improve my work efficiency, allowing me to easily use HTTP requests to write data into the database, optimizing and simplifying many workflow processes.

### 5. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by patryk20120** [comment]
Have you tried JSON response format? 
![image](https://github.com/langgenius/dify/assets/9155931/39496ae6-f5ba-4c4f-95e5-2f24d0b115e8)
